client/a8c-for-agencies/components/items-dashboard/items-dataviews/style.scss

```diff
@@ -0,0 +1,265 @@
+@import "@wordpress/base-styles/breakpoints";
+@import "@wordpress/base-styles/mixins";
+
```

Comment:
The same styles are from the SitesDataViews component. Here it'll be only the common styles